spec/unit/provider/user/useradd_spec.rb

```diff
@@ -385,6 +429,28 @@
       resource[:allowdupe] = :true
       expect(provider.addcmd).not_to include("-o")
     end
+
+    context 'when forcelocal=true' do
+      before do
+        resource[:forcelocal] = :true
+      end
+
+      it 'does not pass lchage options to luseradd for password_max_age' do
+        resource[:password_max_age] = 100
+        expect(provider.addcmd).not_to include('-M')
```

Comment:
I'm a little confused by these tests. You're attempting to manage password max age, but expecting it to not try to manage that resource? Is this because you're explicitely calling `addcmd`?

Reply:
yeah, I wanted to add something that would actually catch the problem.

So addcmd is going to call 'useradd' which should not have any of the options for password age that we'd pass to 'chage'

I should probably add a comment as to why we're doing that

Reply:
Oh cool, yes, thank you for the clarification. This is awesome to have!